.github/actions/checkout-pytorch/action.yml

```diff
@@ -40,6 +40,11 @@ runs:
         fi
         mkdir "${GITHUB_WORKSPACE}"
 
+        # Use all available CPUs for fetching
+        cd "${GITHUB_WORKSPACE}"
+        git config --global fetch.parallel 0
```

Comment:
[The docs](https://git-scm.com/docs/git-config) are surprisingly vague on how many cores it'll actually use:
"A value of 0 will give some reasonable default"